### ISSUE TITLE
How to create an image for arm64 machine?

### ISSUE THREAD

**Issue by yaaaaaao** [fkie-cad/docker_ghidra_headless_base#13]
I encountered a problem when I used the image to build, and an error was displayed indicating that the image does not apply to arm64 architecture. Is there any docker image applicable to arm64 architecture?

**Comment by jstucke** [comment]
Did you try building the image yourself using the `Dockerfile` from the repository? Currently, the image is only pre-built for amd64. We should be able to add arm64 with `docker buildx`, though.

**Comment by yaaaaaao** [comment]
> Did you try building the image yourself using the `Dockerfile` from the repository? Currently, the image is only pre-built for amd64. We should be able to add arm64 with `docker buildx`, though.

Yeah, I solved the problem last week by this solution. Thank you very much!
